docs/reference/Admin_Actions.md

```diff
@@ -203,3 +203,1432 @@ Modified some packages data on the backend database, due to incompatibility with
 - `ReeSilva/atom-homestead` - Removed `v0.10.0`
 - `kraih/atom-perltidy` - Removed `0.1`, `0.2`, `0.3`
 - `swdotcom/swdc-atom` - Removed Package
+
+## Pre-Backend
+
+In the time before the backend was actually created, when the Pulsar team had archived all existing packages from the Atom Package Registry, there were two steps taken that resulted in some packages not making it past the initial migration, and being excluded from the Pulsar Package Registry. Those steps are detailed below.
+
+### Database Migration
+
+During the initial migration of packages to the database, it's possible some packages would break some rules that had been configured within the schema of this new database. If you'd like the script and full logs of this process are available on [`confused-Techie/atom-package-migrator-to-db-2`](https://github.com/confused-Techie/atom-package-migrator-to-db-2/tree/main). Below are the package's that failed to migrate during this process:
```

Comment:
```suggestion
During the initial migration of packages to the database, it's possible some packages would break some rules that had been configured within the schema of this new database. The script and full logs of this process are available at [`confused-Techie/atom-package-migrator-to-db-2`](https://github.com/confused-Techie/atom-package-migrator-to-db-2/tree/main). Below are the packages that failed to migrate during this process:
```